src/core/propertyTypes.js

```diff
@@ -38,10 +39,20 @@ function registerPropertyType (type, defaultValue, parse, stringify) {
 }
 module.exports.registerPropertyType = registerPropertyType;
 
+function arrayParse (value) {
+  return value.length === 0 ?
+    [] :
+    value.split(' ');
+}
+
 function defaultParse (value) {
   return value;
 }
 
+function arrayStringify (value) {
```

Comment:
nitpick but this can sit next to the `arrayParse`, it's a bit alphabetized right now